### PR TITLE
update versions to trigger the build

### PR DESCRIPTION
**Pull request by ElectNewt**
I updated the version number to be able to create the release for nuget as I need the Filters on the `Agent.Services` 

Thanks.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Unreleased
+
+## 1.6.10.4
 * Added the `TaggedAddresses` field to the `AgentService`, `CatalogService` and `AgentServiceRegistration` structures
```
Comment (marcin-krystianc) [inline CHANGELOG.md:6]:
```suggestion
* Added the `TaggedAddresses` field to the `AgentService`, `CatalogService` and `AgentServiceRegistration` structures (#111)
```

**Review by marcin-krystianc — COMMENTED** [review]
@ElectNewt  Thanks for you PR.
I've just suggested some additions to changelog (it is missing some entries).

**Review by marcin-krystianc — APPROVED** [review]

**Comment by marcin-krystianc** [comment]
@ElectNewt FYI, version v1.6.10.4 has just been published. Thanks again for your PR.

**Comment by ElectNewt** [comment]
@marcin-krystianc  thanks, sorry for not seing the changelog changes on time, its a bank holiday in irleand and I've been away from the pc.

Thanks for publishing the veersion.

**Pull request merged**